docs/apm/real-user-monitoring/index.md

```diff
@@ -47,7 +47,7 @@ See Real User Monitoring in action.
 
 Access Traces to confirm that your Sumo Logic service package has been upgraded to include Traces and Real User Monitoring.
 
-[**Classic UI**](/docs/get-started/sumo-logic-ui-classic). To access Traces, click the **+ New** button at the top of the screen and select **Traces**.
+[**Classic UI**](/docs/get-started/sumo-logic-ui-classic). To access Traces, go to your **Home** dashboard and select **Traces**.
```

Comment:
```suggestion
[**Classic UI**](/docs/get-started/sumo-logic-ui-classic). To access Traces, go to the **Home** screen and select **Traces**.
```

Reply:
I suggest globally changing "your **Home** dashboard" to "the **Home** screen". "Dashboard" could be confused with the dashboards functionality, and "your" implies it's a screen specific to the user.

Reply:
Thank you! Much better. I'll make this change globally.